soundbay/conf/augmentations/_augmentations.yaml

```diff
@@ -1,5 +1,12 @@
 # @package _global_
 _augmentations:
+  add_noise:
+    _target_: soundbay.custom_audiomentations.AddMultichannelBackgroundNoise
+    sounds_path:  '/mnt/c/Users/noam/whale/all_brazil_noise'
+    min_snr_in_db: 3
```

Comment:
why do we need those parameters? I didn't find them in the logic code

Reply:
Since this is an augmentation it is applied using `instantiate` in the dataloader.
So it will take all these parameters and apply them using this line in our code
https://github.com/deep-voice/soundbay/blob/892439efb88a9fcd4c4236dc00cae562b484579c/soundbay/data.py#L151

Reply:
yep it's an argument of the base class